Web/Scotty/Action.hs

```diff
@@ -270,16 +277,61 @@ liftAndCatchIO :: MonadIO m => IO a -> ActionT m a
 liftAndCatchIO = liftIO
 {-# DEPRECATED liftAndCatchIO "Use liftIO instead" #-}
 
--- | Redirect to given URL. Like throwing an uncatchable exception. Any code after the call to redirect
--- will not be run.
+-- | Synonym for 'redirect302'.
+-- If you are unsure which redirect to use, you probably want this one.
 --
 -- > redirect "http://www.google.com"
 --
 -- OR
 --
 -- > redirect "/foo/bar"
 redirect :: (Monad m) => T.Text -> ActionT m a
-redirect = E.throw . AERedirect
+redirect = redirect302
+
+-- | Redirect to given URL with status 300. Like throwing
```

Comment:
Perhaps you could add the description string to the status codes? e.g. 300 (Multiple choices), 301 (Moved permanently) etc?  For those who don't know all status codes by heart.

Reply:
Great idea @ocramz! Updated all the doc strings.